frontends/web/src/locales/en/app.json

```diff
@@ -708,6 +708,14 @@
     "wrongKeystore": "Wrong wallet connected. Please make sure to insert the correct device matching this account.",
     "wrongKeystore2": " If you are using the optional passphrase, make sure you have entered the correct passphrase for the account."
   },
+  "exchange": {
+    "buySell": {
+      "coinNotSupported": "No {{action}} options available for this coin type.",
+      "regionNotSupported": "No {{action}} options available for this region.",
+      "updateFirmware": "Update firmware to sell",
+      "updateNow": "Update now"
```

Comment:
no strong opinion, but I think that specifying that the upgrade is needed to sell could be useful. Otherwise could be a bit weird to get a generic fw upgrade request there, imo.

Reply:
we should have a general "A firmware update is required to make use of this feature" text, as we have some instances already and there will be more. Having a specific string for all of them seems overkill.